### PR TITLE
Feature/ci githubactions

### LINKED ISSUE #1289: Moving from travis-ci
Hi, since travis dropped the number of builders for open source projects, builds take too long to complete https://www.traviscistatus.com/#month.

Would you agree on moving to github actions or something else? I can help sending a PR if that's the case :)

### PR DESCRIPTION
**Pull request by unfor19**
- Added GitHub Actions workflow for tests
- Fixed formatting according to black
- Workflow (pipeline) [Logs](https://github.com/unfor19/python-prompt-toolkit/runs/1494486372?check_suite_focus=true)
- Average workflow (pipeline) duration is 3 minutes

- **TODO**: Add the status badge to README.rst - [test](https://github.com/unfor19/python-prompt-toolkit/actions?query=workflow%3Atest)<br>`[test](https://github.com/prompt-toolkit/python-prompt-toolkit/actions?query=workflow%3Atest)`

@jonathanslenders - let me know if you have any questions

Closes #1289

### CODE REVIEW

**Comment by jonathanslenders** [comment]
Hi @unfor19 , I think this looks good. Is there anything I still have to do to activate Github actions and see the pipeline, or should I merge it like this?

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          mypy prompt_toolkit
+          isort -c --profile black prompt_toolkit examples tests setup.py
+          black --check prompt_toolkit examples tests setup.py
+      - name: Validate README.md
+        # Ensure that the README renders correctly (required for uploading to PyPI).        
+        run: |
+          python -m readme_renderer README.rst > /dev/null
+      - name: Run codecov
+        run: |
+          codecov
```
Comment (bjorn3) [inline .github/workflows/test.yaml:44]: nit: missing trailing newline

**Comment by bjorn3** [comment]
Github actions is activated as soon as you merge this I think.

**Comment by unfor19** [comment]
> Hi @unfor19 , I think this looks good. Is there anything I still have to do to activate Github actions and see the pipeline, or should I merge it like this?

Merge it as it is, and it'll do its magic :)

File: .github/workflows/test.yaml (hunk 1 above)
Comment (unfor19) [reply .github/workflows/test.yaml]: added newline

**Comment by unfor19** [comment]
@jonathanslenders ? 🙄

**Comment by jonathanslenders** [comment]
Seems to work. Thanks!

**Comment by unfor19** [comment]
Superb!
I recommend that you cancel the Travis CI integration, it seems like it's super

**Pull request merged**